dev/AnimatedVisualPlayer/TestUI/AnimatedVisualPlayerPage.xaml.cs

```diff
@@ -269,7 +279,7 @@ void OnFrameArrived(Direct3D11CaptureFramePool sender, object args)
             return await tcs.Task;
         }
 
-        private bool IsRS5OrHigher()
+        static bool IsRS5OrHigher()
         {
```

Comment:
Consider using the helper functions in:
dev\CommonManaged\PlatformConfiguration.cs

PlatformConfiguration.IsOsVersionGreaterThanOrEqual(OSVersion.Redstone5)

Reply:
Done.